### PR TITLE
Added verification that BROADR materials are given in ascending order.

### PR DESCRIPTION
**Pull request by nicholas-sly**
Added check and tests.

### CODE REVIEW

File: src/njoy21/input/BROADR.hpp [hunk 1]
```diff
@@ -22,10 +22,20 @@ class BROADR {
       card4( is, this->card2.ntemp2, this->card2.istrap, this->card2.temp1 ){
   	Card5 card5( is );
   	if ( card5.mat1.value ){
+          if( card5.mat1.value < this->card2.mat1.value ){
+            Log::error("BROADR materials must be entered in ascending order." );
```
Comment (jlconlin) [inline src/njoy21/input/BROADR.hpp]: Making this error statement the same as the next one makes it difficult to know which one threw the error. This statement could be a good `Log::info`, but we need more information so we know more specifically where the problem is.

File: src/njoy21/input/GROUPR/Card8b/Wght/test/Wght.test.cpp [hunk 2]
```diff
@@ -16,7 +16,7 @@ SCENARIO( "Wght input values", "[Card8b], [Wght]" ){
     THEN( "the returned TAB1 has the correct values" ){
       iRecordStream<char> iss( ( std::istringstream( sTAB1 ) ) );
 
-      njoy::ENDFtk::UnivariateTabulation TAB1 = argument::extract< 
+      njoy::ENDFtk::TabulationRecord TAB1 = argument::extract<
```
Comment (apmccartney) [inline src/njoy21/input/GROUPR/Card8b/Wght/test/Wght.test.cpp]: git checkout master src/njoy21/input/GROUPR/Card8b/Wght/test/Wght.test.cpp

**Comment by apmccartney** [comment]
I'll come check testing and then merge

File: src/njoy21/input/BROADR.hpp (hunk 1 above)
Comment (jlconlin) [reply src/njoy21/input/BROADR.hpp]: Any comments on this?

**Pull request merged**